.github/workflows/test_dbt_models.yaml

```diff
@@ -1,22 +1,38 @@
 name: test-dbt-models
 
-# This workflow is not scheduled or run on PRs because it is manually triggered
-# by the Data Department's sqoop data extraction process upon completion. See
-# https://github.com/ccao-data/service-sqoop-iasworld
+# This workflow is not configured to run on PRs, because PRs have their own CI
+# process defined by the `build-and-test-dbt` workflow. It runs under two
+# circumstances:
+#
+#   1. When manually triggered by the Data Department's Spark data extraction
+#      process upon completion. This process runs iasWorld data tests. See
+#      https://github.com/ccao-data/service-sqoop-iasworld
+#
+#   2. On the cron schedule set below, which runs non-iasWorld data tests on
+#      a weekly schedule to alert the team of any problems proactively
 on:
   workflow_dispatch:
     inputs:
       select:
         description: >
           Optional space-separated list of tests to run (defaults to all
-          iasWorld data tests)
+          non-iasWorld data tests)
```

Comment:
It's necessary to change our default to non-iasWorld tests in order to run those tests on a schedule, since input variables [will always be null in scheduled workflows](https://stackoverflow.com/a/73495922). This change will require us to update our Spark config to pass the `selector` argument in. We do that in this companion PR: https://github.com/ccao-data/service-spark-iasworld/pull/20